Doc/src/examples.rst

```diff
@@ -87,19 +105,20 @@ As in the first example, we use the EAX mode to allow detection of unauthorized
     from Crypto.Random import get_random_bytes
     from Crypto.Cipher import AES, PKCS1_OAEP
 
+    data = "I met aliens in UFO. Here is the map.".encode("utf-8")
     file_out = open("encrypted_data.bin", "wb")
 
     recipient_key = RSA.import_key(open("receiver.pem").read())
     session_key = get_random_bytes(16)
 
-    # Encrypt the session key with the public RSA key
+    # Encrypt the session key with the public RSA key                                                                                                                                                            
     cipher_rsa = PKCS1_OAEP.new(recipient_key)
-    file_out.write(cipher_rsa.encrypt(session_key))
+    enc_session_key = cipher_rsa.encrypt(session_key)
 
-    # Encrypt the data with the AES session key
+    # Encrypt the data with the AES session key                                                                                                                                                                  
     cipher_aes = AES.new(session_key, AES.MODE_EAX)
     ciphertext, tag = cipher_aes.encrypt_and_digest(data)
-    [ file_out.write(x) for x in (cipher_aes.nonce, tag, ciphertext) ]
+    [ file_out.write(x) for x in (enc_session_key, cipher_aes.nonce, tag, ciphertext) ]
 
 The receiver has the private RSA key. They will use it to decrypt the session key
```

Comment:
Just by looking at this patch, doesnm't this comment contradicts the comment in line 75?

Reply:
which two parts are contradicting to each other?

Reply:
Line 75: The following code generates public key stored in ``receiver.pem`` and private key stored in ``private.pem``.
Line 123: The receiver has the private RSA key

Reply:
Right. Probably two better filenames are `rx_public.pem` and `rx_private.pem`?